src/App.js

```diff
@@ -1,10 +1,19 @@
-import React, { useState } from "react";
+import React, { useState , useEffect} from "react";
 import logo from "./logo.svg";
 import "./App.css";
 import Form from "./Components/Form.js";
+import List from "./Components/List.js";
 
 function App() {
-  const [activities, setActivities] = useState([]);
+
+  const initialActivities = JSON.parse(localStorage.getItem("activities")) || [];
+  const [activities, setActivities] = useState(initialActivities);
+
+  useEffect(() => {
+    localStorage.setItem("activities", JSON.stringify(activities));
+  }, [activities]);
+
```

Comment:
Using the `use-local-storage-state` library in React offers some advantages over using localStorage directly in combination with the useState and useEffect hooks. 

Please take a look in the respective handout: [react with local storage](https://github.com/neuefische/hh-web-23-5/blob/main/sessions/react-with-local-storage/react-with-local-storage.md#use-local-storage-state)🙂

```js
import useLocalStorageState from 'use-local-storage-state';

const [activities, setActivities] = useLocalStorageState('activities', []);
```